mlflow/tensorflow.py

```diff
@@ -694,12 +694,30 @@ def _setup_callbacks(lst):
 def autolog(every_n_iter=100):
     # pylint: disable=E0611
     """
-    Enable automatic logging from TensorFlow to MLflow. If applicable,
-    model checkpoints are logged as artifacts to a 'models' directory, along
-    with any TensorBoard log data.
-
-    Refer to the tracking documentation for
-    information on what is logged with different TensorFlow workflows.
+    Enables automatic logging from TensorFlow to MLflow. This methods logs, when applicable,
```

Comment:
Sam comment as before, ow about "Enables automatic logging from TensorFlow to MLflow. Autologging captures the following information:"